book/website/reproducible-research/rdm/rdm-smp.md

```diff
@@ -0,0 +1,82 @@
+[](#rr-rdm-smp)
```

Comment:
Oh, sorry, this isn't what I meant :sweat_smile:.

You created the labels in the correct way, `(my-label)=`, but when referencing them, the MyST way to do that is `[](#my-label)` or `[custom link text](#my-label)`.

`` {ref}`my-label` `` is the older, Sphinx style.

Reply:
There is information [in the MyST documentation](https://mystmd.org/guide/cross-references) and in [the community handbook](https://book.the-turing-way.org/community-handbook/style/style-crossref)